File: menu/settings_menu.py
def proses_aksi(aksi):
    if aksi == "audio":
        return {
            "menu": "settings",
            "fitur": "audio",
            "status": "Fitur Audio masih dalam tahap pengembangan."
        }

    if aksi == "grafis":
        return {
            "menu": "settings",
            "fitur": "grafis",
            "status": "Fitur Grafis masih dalam tahap pengembangan."
        }

    return {
        "menu": "settings",
        "status": "Pengaturan tidak dikenal."
    }

File: static/js/settings_menu.js
function bukaSettings() {
    document.getElementById('menu-utama').classList.add('hidden');
    document.getElementById('menu-settings').classList.remove('hidden');

    tampilkanPesan("Silakan pilih pengaturan.");
}


function pilihSubSettings(namaFitur) {
    tampilkanPesan(
        `Pengaturan ${namaFitur} masih dalam tahap pengembangan.`
    );
}


function kembaliKeMenuUtama() {
    // Sembunyikan Settings
    document.getElementById('menu-settings').classList.add('hidden');

    // Sembunyikan menu game
    document.getElementById('menu-game').classList.add('hidden');

    // Sembunyikan prolog
    document.getElementById('prolog').classList.add('hidden');

    // Tampilkan menu utama
    document.getElementById('menu-utama').classList.remove('hidden');

    // Bersihkan pesan
    tampilkanPesan("");
}
